runners/direct-java/src/main/java/org/apache/beam/runners/direct/RootProviderRegistry.java

```diff
@@ -28,25 +28,38 @@
 import org.apache.beam.runners.core.construction.PTransformTranslation;
 import org.apache.beam.sdk.options.PipelineOptions;
 import org.apache.beam.sdk.runners.AppliedPTransform;
+import org.apache.beam.sdk.transforms.Impulse;
 import org.apache.beam.sdk.transforms.PTransform;
 
 /**
  * A {@link RootInputProvider} that delegates to primitive {@link RootInputProvider} implementations
  * based on the type of {@link PTransform} of the application.
  */
 class RootProviderRegistry {
-  public static RootProviderRegistry defaultRegistry(
+  /**
+   * Returns a {@link RootProviderRegistry} that supports the Java SDK root transforms.
+   */
+  public static RootProviderRegistry javaNativeRegistry(
       EvaluationContext context, PipelineOptions options) {
-    ImmutableMap.Builder<String, RootInputProvider<?, ?, ?>> defaultProviders =
-        ImmutableMap.builder();
-    defaultProviders
-        .put(IMPULSE_TRANSFORM_URN, new ImpulseEvaluatorFactory.ImpulseRootProvider(context))
-        .put(
-            PTransformTranslation.READ_TRANSFORM_URN,
-            ReadEvaluatorFactory.inputProvider(context, options))
-        .put(DIRECT_TEST_STREAM_URN, new TestStreamEvaluatorFactory.InputProvider(context))
-        .put(FLATTEN_TRANSFORM_URN, new EmptyInputProvider());
-    return new RootProviderRegistry(defaultProviders.build());
+    return new RootProviderRegistry(
+        ImmutableMap.<String, RootInputProvider<?, ?, ?>>builder()
+            .put(IMPULSE_TRANSFORM_URN, new ImpulseEvaluatorFactory.ImpulseRootProvider(context))
+            .put(
+                PTransformTranslation.READ_TRANSFORM_URN,
+                ReadEvaluatorFactory.inputProvider(context, options))
+            .put(DIRECT_TEST_STREAM_URN, new TestStreamEvaluatorFactory.InputProvider(context))
+            .put(FLATTEN_TRANSFORM_URN, new EmptyInputProvider())
+            .build());
+  }
+
+  /**
+   * Returns a {@link RootProviderRegistry} that only supports the {@link Impulse} primitive.
+   */
+  public static RootProviderRegistry impulseRegistry(EvaluationContext ctxt) {
```

Comment:
This file has inconsistent uses of "ctxt" vs. "context". I would stick to one of them.

Reply:
Done.